### PR TITLE
feat: unified mgnt of mongo indexes

### PR DESCRIPTION
**Pull request by wildeslam**

### CODE REVIEW

File: arex-storage-web-api/src/main/java/com/arextest/storage/beans/IndexesSettingConfiguration.java [hunk 1]
```diff
@@ -43,20 +41,17 @@ private void ensureMockerQueryIndex(MongoDatabase database) {
                 AREXMocker.class);
 
         try {
-          Document index = new Document();
-          index.append(AREXMocker.Fields.recordId, 1);
-          collection.createIndex(index);
+          collection.dropIndexes();
```
Comment (coryhh) [inline arex-storage-web-api/src/main/java/com/arextest/storage/beans/IndexesSettingConfiguration.java:44]: What is the purpose of this method？
Delete all indexes first and then recreate them?

File: arex-storage-web-api/src/main/java/com/arextest/storage/beans/IndexesSettingConfiguration.java [hunk 2]
```diff
@@ -22,7 +20,7 @@
 
 @Slf4j
 @Configuration(proxyBeanMethods = false)
-public class IndexsSettingConfiguration {
+public class IndexesSettingConfiguration {
```
Comment (coryhh) [inline arex-storage-web-api/src/main/java/com/arextest/storage/beans/IndexesSettingConfiguration.java:23]: How to solve the update ttl index scenario？

File: arex-storage-web-api/src/main/java/com/arextest/storage/beans/IndexesSettingConfiguration.java (hunk 1 above)
Comment (wildeslam) [reply arex-storage-web-api/src/main/java/com/arextest/storage/beans/IndexesSettingConfiguration.java:44]: image
There have been operations to delete specific indexes before, which cause lots of error logs. Now it has been changed to delete all indexes.

File: arex-storage-web-api/src/main/java/com/arextest/storage/beans/IndexesSettingConfiguration.java (hunk 2 above)
Comment (wildeslam) [reply arex-storage-web-api/src/main/java/com/arextest/storage/beans/IndexesSettingConfiguration.java:23]: Need to add drop indexes logic.

**Review by coryhh — APPROVED** [review]

**Review by QizhengMo — DISMISSED** [review]

**Review by QizhengMo — DISMISSED** [review]

**Review by binyu1005 — DISMISSED** [review]

**Review by coryhh — APPROVED** [review]

**Review by QizhengMo — DISMISSED** [review]

**Review by coryhh — APPROVED** [review]

**Review by coryhh — APPROVED** [review]

**Review by binyu1005 — APPROVED** [review]

**Pull request merged**